ReSharper.FSharp/src/FSharp.Psi/src/FSharp.psi

```diff
@@ -1120,6 +1120,8 @@ interface fSharpTypeMemberDeclaration:
   letBindingsDeclaration |
   overridableMemberDeclaration |
   doStatement |
+  accessorDeclaration |
```

Comment:
Why is it needed?

Reply:
At some point I needed it, but now I don't need it, I will delete